pkg/collector/k8s_api.go

```diff
@@ -114,19 +116,35 @@ func NewK8sAPICollector(ctx context.Context, cfg *config.KubehoundConfig) (Colle
 	}
 
 	return &k8sAPICollector{
-		cfg:       cfg.Collector.Live,
-		clientset: clientset,
-		log:       l,
-		rl:        ratelimit.New(cfg.Collector.Live.RateLimitPerSecond), // per second
-		tags:      *newCollectorTags(),
-		waitTime:  map[string]time.Duration{},
-		startTime: time.Now(),
-		mu:        &sync.Mutex{},
+		cfg:         cfg.Collector.Live,
+		clientset:   clientset,
+		log:         l,
+		rl:          ratelimit.New(cfg.Collector.Live.RateLimitPerSecond), // per second
+		tags:        *newCollectorTags(),
```

Comment:
same here, do we need a pointer to this instead of just returning the res? :shrug:

Reply:
fixed